CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing to pgcopydb
+
+We're happy you want to contribute! You can help us in different ways:
+
+* Open an [issue](https://github.com/dimitri/pgcopydb/issues) with suggestions
+  for improvements, potential bugs, etc.
+* Fork this repository and submit a pull request
+
+### Building from source code
+
+Follow the relevant docs at our
+[README.md](https://github.com/dimitri/pgcopydb/#installing-pgcopydb)
```

Comment:
I think the README should be as short as possible, and now that we have build/install coverage in the main documentation at https://pgcopydb.readthedocs.io/en/latest/install.html#build-from-sources I'm wondering if we should remove the section from the README, and in this PR point to the docs. What do you think?

Reply:
I read through that piece of the README, and realized that yum installation is outdated for a while. I think it is a good idea to remove installation instructions completely, and add a reference to our docs.

I also took the liberty of adding a reference in our docs for building from source to CONTRIBUTING.md.